interpreter/cling/lib/Interpreter/CIFactory.cpp

```diff
@@ -570,6 +570,9 @@ namespace {
     llvm::SmallString<256> servIncLoc(getIncludePathForHeader(HS, "windows.h"));
 #endif
     llvm::SmallString<128> cIncLoc(getIncludePathForHeader(HS, "time.h"));
+    // FIXME: Diagnose this until we teach cling how to work without libc.
+    if (!llvm::sys::fs::exists(cIncLoc))
+      llvm::errs() << "glibc must be installed.\b";
```

Comment:
How do I install glibc on Windows? ;-) And why `\b` (aka backspace)?

```suggestion
      llvm::errs() << "C system headers (glibc / Xcode / Windows SDK) must be installed.\n";
```